telegram/_bot.py

```diff
@@ -848,11 +868,23 @@ async def send_message(
             :class:`telegram.Message`: On success, the sent message is returned.
 
         Raises:
-            :class:`telegram.error.TelegramError`
+            :exc:`ValueError`: If both :paramref:`disable_web_page_preview` and
+                :paramref:`link_preview_options` are passed.
+            :class:`telegram.error.TelegramError`: For other errors.
 
         """
         data: JSONDict = {"chat_id": chat_id, "text": text, "entities": entities}
 
+        if not isinstance(disable_web_page_preview, DefaultValue) and not isinstance(
+            link_preview_options, DefaultValue
+        ):
+            raise ValueError(
+                "`disable_web_page_preview` and `link_preview_options` are mutually  exclusive."
+            )
+        # Convert to LinkPreviewOptions:
+        if not isinstance(disable_web_page_preview, DefaultValue):
+            link_preview_options = LinkPreviewOptions(is_disabled=disable_web_page_preview)
+
```

Comment:
@harshil21 would a helper function be worth the while?

Reply:
hmm so far we only have 2 instances. If another one comes up, it's worth

Reply:
I think there was one in `InputMedia*`?